doc/container-environment.md

```diff
@@ -0,0 +1,84 @@
+# Container runtime environment
+LXD attempts to present a consistent environment to the container it runs.
+
+The exact environment will differ slightly based on kernel features and  
+user configuration but will otherwise be identical for all containers.
+
+## PID1
+LXD spawns whatever is located at /sbin/init as the initial process of the container (PID 1).
+This binary should act as a proper init system, including handling re-parented processes.
+
+LXD's communication with PID1 in the container is limited to two signals:
+ - SIGINT (or alternatively SIGRTMIN+3) to trigger a reboot of the container
+ - SIGPWR (or to trigger a clean shutdown of the container
```

Comment:
This section is incorrect. :) It should be:
```
- SIGINT to trigger a reboot of the container
- SIGPWR (or alternatively SIGRTMIN+3) to trigger a clean shutdown of the container
```

(There's also a misplaced bracket in there. :))